sale_fixed_discount/models/sale_order_line.py

```diff
@@ -15,6 +15,14 @@ class SaleOrderLine(models.Model):
         help="Fixed amount discount.",
     )
 
+    @api.model_create_multi
```

Comment:
@sefirosweb 

You don't need to override create():

```suggestion
    @api.depends("discount_fixed")
    def _compute_discount(self):
        super()._compute_discount()
        for line in self:
            if line.discount_fixed:
                line.discount = line._get_discount_from_fixed_discount()
```

And then, remove onchange.

Reply:
Yes you are right, I test it and works for me